data/json/recipes/food/brewing.json

```diff
@@ -267,24 +256,56 @@
   },
   {
     "type": "recipe",
-    "activity_level": "LIGHT_EXERCISE",
+    "activity_level": "NO_EXERCISE",
     "result": "brew_pine_wine",
     "result_mult": 3,
     "category": "CC_FOOD",
     "subcategory": "CSC_FOOD_BREW",
     "skill_used": "cooking",
     "difficulty": 3,
-    "time": "80 m",
-    "batch_time_factors": [ 50, 4 ],
+    "time": "30 m",
+    "batch_time_factors": [ 95, 4 ],
     "book_learn": [ [ "survival_book", 3 ], [ "textbook_survival", 3 ], [ "manual_survival", 3 ] ],
-    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "BOIL", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
     "components": [
       [ [ "water", 3 ], [ "water_clean", 3 ] ],
       [ [ "pine_bough", 5 ] ],
       [ [ "sugar", 10 ], [ "honeycomb", 1 ], [ "honey_bottled", 2 ], [ "honey_glassed", 2 ], [ "syrup", 2 ] ],
       [ [ "yeast", 1 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "malting_grain",
+    "result_mult": 1,
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREW",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "30 s",
+    "batch_time_factors": [ 95, 4 ],
+    "book_learn": [ [ "brewing_cookbook", 4 ] ],
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "components": [ [ [ "water", 1 ], [ "water_clean", 1 ] ], [ [ "corn", 1 ], [ "barley", 1 ], [ "buckwheat", 1 ], [ "oats", 1 ] ] ]
```

Comment:
Please add "wheat" to the possible grains.

Reply:
Added